apps/core/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "@kiwicom/orbit-design-tokens": "^0.2.5",
-    "@kiwicom/universal-components": "^0.0.6",
+    "@kiwicom/universal-components": "^0.0.7-alpha.5",
```

Comment:
`ConnectedCard` component is available from version `0.0.7`, but there is some bug with `DEVICE_LOCALE` in `node_modules/@kiwicom/universal-components/lib/native/utils/DateUtils/DateUtils.js`. This issue is also connected with test failures. 

What can we do with that?

Reply:
Fixed in new version `0.0.7`